Marlin/pins_RAMPS_FD.h

```diff
@@ -18,20 +18,20 @@
 #define X_STEP_PIN         63
 #define X_DIR_PIN          62
 #define X_ENABLE_PIN       48
-#define X_MIN_PIN          22
-#define X_MAX_PIN          2
+#define X_MIN_PIN          -1 //22
```

Comment:
Yes,
In delta configuration the x_min and y_min is not use because a only pin user for a bed leveling is z_min

You looks a pins for Ramps_FD 1.4RevA on this link 
https://drive.google.com/file/d/0B6mn--RUVyiYM3VKcGMybUh6eTg/view?usp=sharing